challenges/2017-08-10-radio-transmitters/README.md

```diff
@@ -0,0 +1,7 @@
+# Radio Transmitters
+
+Link: (Radio Transmitters)[https://www.hackerrank.com/challenges/hackerland-radio-transmitters/problem]
```

Comment:
To keep the pattern, could you put the link right below the description, in the format?

[Source](https://www.hackerrank.com/challenges/hackerland-radio-transmitters/problem)